lib/topological_inventory/ansible_tower/operations/core/ansible_tower_client.rb

```diff
@@ -0,0 +1,129 @@
+require "topological_inventory/ansible_tower/logging"
+require "topological_inventory/ansible_tower/connection"
+require "topological_inventory/ansible_tower/operations/core/sources_api_client"
+require "topological_inventory/ansible_tower/operations/core/topology_api_client"
+
+module TopologicalInventory
+  module AnsibleTower
+    module Operations
+      module Core
+        class AnsibleTowerClient
+          include Logging
+          include Core::TopologyApiClient
+
+          attr_accessor :connection_manager
+
+          SLEEP_POLL = 10
+          POLL_TIMEOUT = 1800
+
+          def initialize(source_id, task_id, identity = nil)
+            self.identity   = identity
+            self.source_id  = source_id
+            self.task_id    = task_id
+
+            self.connection_manager = TopologicalInventory::AnsibleTower::Connection.new
```

Comment:
This isn't really a connection manager like openshift has (and would argue it doesn't need to be, ansible_tower is much less complex than what openshift has to deal with).